examples/viewer_lib/ui/segmentation/segment_rendering_ui.py

```diff
@@ -0,0 +1,105 @@
+from dataclasses import dataclass, field
+
+from trame_server.utils.typed_state import TypedState
+from trame_vuetify.widgets.vuetify3 import (
+    Template,
+    VBtn,
+    VBtnToggle,
+    VCard,
+    VCardItem,
+    VCardText,
+    VSlider,
+)
+
+from trame_slicer.segmentation import SegmentationOpacityEnum
+
+from ..utils import ControlButton, FlexContainer, Text
+
+
+@dataclass
+class SegmentRenderingState:
```

Comment:
Rendering is usually reserved for the actual GPU / window rendering.

For Slicer, it's better to rename to `SegmentDisplayState` (closely related to the SegmentationNodeDisplay node)